composer.json

```diff
@@ -14,13 +14,24 @@
     }
   ],
   "autoload": {
-    "psr-0": {
-      "PAMI": "src/mg"
+    "psr-4": {
+      "PAMI\\": "src/mg/PAMI"
     }
   },
   "require": {
     "php": ">=5.3.3",
     "apache/log4php": "2.3.0",
     "psr/log": "~1.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "4.8.*",
+    "phing/phing": "2.*",
+    "squizlabs/php_codesniffer": "2.*",
+    "sebastian/phpcpd": "*",
```

Comment:
I'd agree with regular depencies.. but in this case these tools are not used *programatically* from the code but *run* to analyse the code itself, so we *should* be ok (they are also only required for the dev environment). The idea is to actually be updated to the latest version in case the new versions are faster or better.